### PR TITLE
chore(config): added commented values to ttk values file

### PR DESCRIPTION
**Pull request by vijayg10**

### CODE REVIEW

File: ml-testing-toolkit/values.yaml [hunk 1]
```diff
@@ -4,47 +4,111 @@ global: {}
...
+  #       host: testing-toolkit.localdomain
+  #       port: 5050
+  #       paths: ['/api/', '/socket.io/']
   config:
     system_config.json:
       HOSTING_ENABLED: false
       OAUTH:
         AUTH_ENABLED: false
-      CONNECTION_MANAGER_AUTH_ENABLED: false
+      CONNECTION_MANAGER:
```
Comment (mdebarros) [inline ml-testing-toolkit/values.yaml:23]: Is this not a breaking change?

I.e. requires a major version bump to the helm chart?

**Review by mdebarros — CHANGES_REQUESTED** [review]
See my comment.

File: ml-testing-toolkit/values.yaml (hunk 1 above)
Comment (vijayg10) [reply ml-testing-toolkit/values.yaml:23]: No Miguel, that param is related to hosted mode and left in old syntax from long time. So we can consider this as a minor fix and don't need major version bump.

Comment (vijayg10) [reply ml-testing-toolkit/values.yaml:23]: May be we can change minor version of TTK helm chart

**Review by mdebarros — APPROVED** [review]

**Pull request merged**